easybuild/easyconfigs/b/bwidget/bwidget-1.10.1-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'Tarball'
+
+name = 'bwidget'
+version = '1.10.1'
+
+homepage = 'https://core.tcl-lang.org/bwidget/home'
+description = 'The BWidget Toolkit is a high-level Widget Set for Tcl/Tk built using native Tcl/Tk 8.x namespaces.'
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+source_urls = ['https://downloads.sourceforge.net/project/tcllib/BWidget/%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['4aea02f38cf92fa4aa44732d4ed98648df839e6537d6f0417c3fe18e1a34f880']
+
+builddependencies = [('binutils', '2.42')]
+
+dependencies = [('Tk', '8.6.14')]
+
+modextrapaths = {'TCLLIBPATH': {'paths': '', 'delimiter': ' '}}
```

Comment:
note for reviewers: this is an EB5 feature, so should not be merged until develop is targeting EB5 (or retarget against the EB5 branch)

Reply:
i swapped it